src/NUnitFramework/nunitlite.tests/TextUIReportTests.cs

```diff
@@ -101,7 +101,13 @@ public void TestsNotRunTest()
                 "Tests Not Run\n\n" +
                 "1) Ignored : NUnit.Tests.Assemblies.MockTestFixture.IgnoreTest\n" +
                 "Ignore Message\n\n" +
-                "2) Explicit : NUnit.Tests.Assemblies.MockTestFixture.ExplicitTest\n\n"));
+                "2) Ignored : NUnit.Tests.Assemblies.MockTestFixture.SkippedTest(1)\n"+
+                "Ignore testcase\n\n"+
+                "3) Ignored : NUnit.Tests.Assemblies.MockTestFixture.SkippedTest(2)\n"+
+                "Ignore testcase\n\n"+
+                "4) Ignored : NUnit.Tests.Assemblies.MockTestFixture.SkippedTest(3)\n"+
+                "Ignore testcase\n\n"+
```

Comment:
Are we here (implicitly) testing the ordering of attributes on a method? We often remind users that the attributes are unordered, so one cannot depend on the ordering. Should we use a TestCaseSource to make the ordering deterministic? (although this will be a bit more verbose)